### ISSUE TITLE
NTR: Moisture in external ear canal by history (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7815]
New term request Moisture in external ear canal by history (segal_210901145824): Swimming, warm humid weather

**Comment by pnrobinson** [comment]
@MickeySegal

this terms seems to refer to something like swimmer's ear rather than the moisture that occurs due to middle ear infection?

**Comment by MickeySegal** [comment]
It refers to the moisture from being immersed in water that predisposes to Swimmer's Ear.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000773
- label: Recent history of moisture in external ear canal
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: PMID:37261805
- def: A recent history of being exposed to moisture in the external ear canal.
- comment: Swimmer's ear may occur when excess moisture in the ear canal allows bacteria or fungi to grow more easily. Risk factors include swimming and use of ear buds while exercising (and sweating).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7815
- synonyms:
